apps/src/code-studio/header.js

```diff
@@ -502,4 +502,17 @@ header.updateTimestamp = function () {
   }
 };
 
+// TODO i18n
+header.showProjectSaveError = () => {
+  const saveErrorTooltip = "It looks like we couldn't save your progress. Make sure you have a " +
+    "good internet connection and try running the project again to save it.";
+
+  const saveErrorHtml = `
+<span class="save_error" title="${saveErrorTooltip}">
```

Comment:
Nit: [Our styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#css) says we follow [thoughtbot's SASS styleguide](https://github.com/thoughtbot/guides/tree/master/style/sass) for CSS, which says:

> Use hyphens when naming mixins, extends, classes, functions & variables: `span-columns` not `span_columns` or `spanColumns`.

I bet this is hard to enforce via linter, given how often classnames and such are dynamic or inline.

Reply:
ahh, that's good to know! I've always chosen arbitrarily.